bin/install.sh

```diff
@@ -11,7 +11,7 @@ hash npm &> /dev/null && npm install || {
 }
 
 echo "Looking for Alfred Preferences..."
-paths=("$1" "$HOME/Dropbox" "$HOME/Library/Mobile Documents/com~apple~CloudDocs" "$HOME/Google Drive" "$HOME/Library/Application Support/Alfred 3" "$HOME/Library/Application Support/Alfred 2")
+paths=("$1" "$HOME/Dropbox" "$HOME/Library/Mobile Documents/com~apple~CloudDocs" "$HOME/Google Drive" "$HOME/Library/Application Support/Alfred 3" "$HOME/Library/Application Support/Alfred 2" "$HOME/Library/Application Support/Alfred")
```

Comment:
In order to default to the latest version, the `Alfred 4` path should come before the Alfred 3/2 paths in the event the user has both versions of the app installed.

Reply:
Fixed in my latest push.

Reply:
Awesome! Thanks for the PR